doc/source/examples/technology_showcase_examples/techdemo-28/ex_28-tecfricstir.rst

```diff
@@ -569,8 +569,12 @@ The following contact settings are used for the ``CONTA174`` elements:
         mapdl.esel('s', 'mat', '', 2)
         mapdl.esel("r", "ename", "", elem)
         esurf = mapdl.mesh._grid.linear_copy().extract_surface().clean()
-        pl.add_mesh(esurf, show_edges=True, show_scalar_bar=False,
-                   style='surface', color=color)
+        if mapdl.mesh.n_elem == 1:
+            # pl.add_points(esurf)
+            pass
```

Comment:
Why was this added?

Reply:
Surprisingly, in the first implementation of the tech demo you could add a point using ``add_mesh``.

After the changes in ``mapdl.mesh`` (and probably some changes in Pyvista), this was showing a warning which broke the CICD until (after a lot of time testing) I fix it.

I didn't delete because I kept fixing other stuff after. Soooo... we can remove it.